mongo/client.go

```diff
@@ -328,6 +328,26 @@ func (c *Client) configure(opts *options.ClientOptions) error {
 
 	// TODO(GODRIVER-814): Add tests for topology, server, and connection related options.
 
+	// Server API options
```

Comment:
As discussed offline, this needs to happen pretty high up in `configure` because it sets `c.serverAPI`, which is referenced by other sections. Can you extend this comment to reflect that so future authors are aware of it?

Reply:
Sure; we wouldn't want someone moving this block around. Added a more nuanced comment.